packages/core/admin/admin/src/pages/Webhooks/ListView/index.js

```diff
@@ -276,47 +239,161 @@ function ListView() {
-          <ListButton>{canCreate && <Button {...omit(newButtonProps, 'Component')} />}</ListButton>
-        </div>
-      )}
+        />
+
+        <ContentLayout>
+          <>
+            {rowsCount > 0 ? (
+              <Table
+                colCount={5}
+                rowCount={webhooks.length}
+                footer={(
+                  <TFooter
+                    onClick={() => (canCreate ? handleGoTo('create') : {})}
+                    icon={<AddIcon />}
+                  >
+                    {formatMessage({ id: 'Settings.webhooks.list.field.add' })}
+                  </TFooter>
+                )}
+              >
+                <Thead>
+                  <Tr>
+                    <Th>
+                      <BaseCheckbox
+                        aria-label={formatMessage({
+                          id: 'Settings.webhooks.list.all-entries.select',
+                        })}
+                        indeterminate={
+                          webhooksToDelete.length > 0 && webhooksToDelete.length < webhooks.length
+                        }
+                        value={webhooksToDelete.length === webhooks.length}
+                        onValueChange={handleSelectAllCheckbox}
+                      />
+                    </Th>
+                    <Th>
+                      <TableLabel>
+                        {formatMessage({ id: 'Settings.webhooks.form.name' })}
+                      </TableLabel>
+                    </Th>
+                    <Th>
+                      <TableLabel>{formatMessage({ id: 'Settings.webhooks.form.url' })}</TableLabel>
+                    </Th>
+                    <Th width="30%">
+                      <TableLabel>
+                        {formatMessage({ id: 'Settings.webhooks.list.th.status' })}
+                      </TableLabel>
+                    </Th>
+                    <Th>
+                      <VisuallyHidden>
+                        {formatMessage({ id: 'Settings.webhooks.list.th.actions' })}
+                      </VisuallyHidden>
+                    </Th>
+                  </Tr>
+                </Thead>
+                <Tbody>
+                  {webhooks.map(webhook => (
+                    <Tr key={webhook.id}>
+                      <Td>
+                        <BaseCheckbox
+                          aria-label={`${formatMessage({ id: 'Settings.webhooks.list.select' })} ${
+                            webhook.name
+                          }`}
+                          value={webhooksToDelete?.includes(webhook.id)}
+                          onValueChange={value => handleSelectOneCheckbox(value, webhook.id)}
+                          id={`${formatMessage({ id: 'Settings.webhooks.list.select' })} ${
+                            webhook.id
+                          }`}
+                          name={`${formatMessage({ id: 'Settings.webhooks.list.select' })} ${
+                            webhook.name
+                          }`}
```

Comment:
The `id` and `name` aren't used to be understood by the user. We should use static `names` and `ids` not translated messages.